internal/store/ingress_test.go

```diff
@@ -231,7 +233,42 @@ func TestIngressStore(t *testing.T) {
 				kube_ingress_created{namespace="ns7",ingress="ingress7"} 1.501569018e+09
 				kube_ingress_metadata_resource_version{namespace="ns7",ingress="ingress7"} 123456
 				`,
-			MetricNames: []string{"kube_ingress_info", "kube_ingress_metadata_resource_version", "kube_ingress_created", "kube_ingress_labels", "kube_ingress_path", "kube_ingress_tls"},
+			MetricNames: []string{"kube_ingress_info", "kube_ingress_metadata_resource_version", "kube_ingress_created", "kube_ingress_labels", "kube_ingress_path", "kube_ingress_tls", "kube_ingress_status"},
+		},
+		{
+			Obj: &networkingv1.Ingress{
+				ObjectMeta: metav1.ObjectMeta{
+					Name:              "ingress8",
+					Namespace:         "ns8",
+					CreationTimestamp: metav1StartTime,
+					ResourceVersion:   "123456",
+				},
+				Status: networkingv1.IngressStatus{
+					LoadBalancer: networkingv1.IngressLoadBalancerStatus{
+						Ingress: []networkingv1.IngressLoadBalancerIngress{
+							{
+
+								IP:       "1.2.3.4",
+								Hostname: "www.example.com",
+								Ports: []networkingv1.IngressPortStatus{
+									{
+										Port:     8888,
+										Protocol: "TCP",
+										Error:    nil,
```

Comment:
Should we expose this error somehow as well? 

I assume it can potentially cause high cardinality, but it will be interesting to know if there's an error on the IngressPortStatus?